2024-03-15 15:07:21.605250+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "MIGRATION_MODEL" not found (this database is empty); SQL statement:
SELECT ID, VERSION FROM MIGRATION_MODEL ORDER BY UPDATE_TIME DESC [42104-224]
2024-03-15 15:07:22.774250+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42104-224]
2024-03-15 15:07:23.001250+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOGLOCK" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOGLOCK [42104-224]
2024-03-15 15:07:23.034251+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" non trouvée
Table "DATABASECHANGELOG" not found; SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42102-224]
2024-03-15 15:20:48.288020+01:00 database: flush
org.h2.message.DbException: Erreur générale: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]"
General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Erreur générale: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]"
General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:200)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:172)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:430)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2024-03-15 15:20:48.291991+01:00 database: opening C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db"
IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db" [90028-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db"
IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db" [90028-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:200)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:172)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:430)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2024-03-15 15:20:48.299988+01:00 database: opening C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb
org.h2.message.DbException: Une erreur est survenue lors de l'ouverture de la base de données: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:334)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Une erreur est survenue lors de l'ouverture de la base de données: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:516)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep0(Native Method)
	at java.base/java.lang.Thread.sleep(Thread.java:509)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 17 more
2024-03-15 15:21:35.827174+01:00 database: flush
org.h2.message.DbException: Erreur générale: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]"
General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Erreur générale: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]"
General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:200)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:172)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:430)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2024-03-15 15:21:35.831137+01:00 database: opening C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db"
IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db" [90028-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db"
IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db" [90028-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:200)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:172)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:430)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2024-03-15 15:21:35.839164+01:00 database: opening C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb
org.h2.message.DbException: Une erreur est survenue lors de l'ouverture de la base de données: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:334)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Une erreur est survenue lors de l'ouverture de la base de données: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:516)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep0(Native Method)
	at java.base/java.lang.Thread.sleep(Thread.java:509)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 17 more
2024-03-15 15:23:06.167011+01:00 database: flush
org.h2.message.DbException: Erreur générale: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]"
General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Erreur générale: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]"
General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:200)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:172)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:430)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2024-03-15 15:23:06.171977+01:00 database: opening C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db"
IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db" [90028-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db"
IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db" [90028-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:200)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:172)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:430)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2024-03-15 15:23:06.179177+01:00 database: opening C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb
org.h2.message.DbException: Une erreur est survenue lors de l'ouverture de la base de données: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:334)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Une erreur est survenue lors de l'ouverture de la base de données: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:516)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep0(Native Method)
	at java.base/java.lang.Thread.sleep(Thread.java:509)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 17 more
2024-03-15 15:23:57.500829+01:00 database: flush
org.h2.message.DbException: Erreur générale: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]"
General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Erreur générale: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]"
General error: "org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 23 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 17 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:200)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:172)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:430)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2024-03-15 15:23:57.507824+01:00 database: opening C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb
org.h2.message.DbException: IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db"
IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db" [90028-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:168)
	at org.h2.mvstore.db.Store.<init>(Store.java:142)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db"
IO Exception: "C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db" [90028-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 18 more
Caused by: org.h2.mvstore.MVStoreException: Could not open file C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb.mv.db [2.2.224/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:126)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	... 15 more
Caused by: java.nio.channels.ClosedByInterruptException
	at java.base/java.nio.channels.spi.AbstractInterruptibleChannel.end(AbstractInterruptibleChannel.java:200)
	at java.base/sun.nio.ch.FileChannelImpl.endBlocking(FileChannelImpl.java:172)
	at java.base/sun.nio.ch.FileChannelImpl.size(FileChannelImpl.java:430)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:120)
	... 19 more
2024-03-15 15:23:57.518825+01:00 database: opening C:/Users/LENOVO/Downloads/keycloak-24.0.1/keycloak-24.0.1/data/h2/keycloakdb
org.h2.message.DbException: Une erreur est survenue lors de l'ouverture de la base de données: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.store.FileLock.getExceptionFatal(FileLock.java:429)
	at org.h2.store.FileLock.sleep(FileLock.java:424)
	at org.h2.store.FileLock.lockFile(FileLock.java:334)
	at org.h2.store.FileLock.lock(FileLock.java:110)
	at org.h2.engine.Database.<init>(Database.java:317)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:322)
	at io.agroal.pool.ConnectionFactory.createConnection(ConnectionFactory.java:232)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:536)
	at io.agroal.pool.ConnectionPool$CreateConnectionTask.call(ConnectionPool.java:517)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at io.agroal.pool.util.PriorityScheduledExecutor.beforeExecute(PriorityScheduledExecutor.java:75)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: Une erreur est survenue lors de l'ouverture de la base de données: "Sleep interrupted"
Error opening database: "Sleep interrupted" [8000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:516)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 20 more
Caused by: java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep0(Native Method)
	at java.base/java.lang.Thread.sleep(Thread.java:509)
	at org.h2.store.FileLock.sleep(FileLock.java:422)
	... 17 more
